README.md

```diff
@@ -1 +1,88 @@
 # Peridio Daemon
+
+## Configuring
+
+Peridiod is configured via a json formatted file on the filesystem. The location of the file defaults to `$XDG_CONFIG_HOME/peridio/peridio-config.json`. if `$XDG_CONFIG_HOME` is not set the default path is `$HOME/.config/peridio/peridio-config.json`. This file location can be overwritten by setting `PERIDIO_CONFIG_FILE=/path/to/peridio.json`. The peridiod configuration has the following top level keys:
+
+* `version`: The configuration version number. Currently this is 1.
+* `device_api`: Configuration for the device api endpoint
+  * `certificate_path`: Path to the device api ca certificate.
+  * `url`: The peridio server device api URL.
+  * `verify`: Enable client side ssl verification for device api connections.
+* `fwup`: Keys related to the use of fwup for the last mile.
+  * `devpath`: The block storage device path to use for applying firmware updates.
+  * `public_keys`: A list of authorized public keys used when verifying update archives.
+* `remote_shell`: Enable or disable the remote console feature.
+* `node`: Node configuration settings
+  * `key_pair_source`: Options are `file`, `uboot-env`, `pkcs11`. This determines the source of the identity key information.
+  * `key_pair_config`: Different depending on the `key_pair_source`
+  When `key_pair_source: file`:
+    * `private_key_path`: Path on the filesystem to a PEM encoded private key file.
+    * `certificate_path`: Path on the filesystem to a PEM encoded x509 certificate file.
+  When `key_pair_source: uboot-env`:
+    * `private_key`: The key in the uboot environment which contains a PEM encoded private key.
+    * `certificate`: The key in the uboot environment which contains a PEM encoded x509 certificate.
+  When `key_pair_source: pkcs11`:
```

Comment:
These "When `` :" blocks are getting rendered not as you probably want.

Reply:
Fixed
<img width="1236" alt="Screenshot 2023-05-17 at 1 23 03 PM" src="https://github.com/peridio/peridiod/assets/1391351/b27e304a-fa59-4e97-8524-d04b950280b9">